packages/components/utils/src/Portal/Portal.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Portal'
+type: 'component'
+status: 'beta'
+slug: /utils/portal/
+typescript: ./Portal.tsx
+---
+
+import { Props } from '@contentful/f36-docs-utils';
+
+`Portal` component can be used to render children in to a DOM node outside of the parent component. It can be useful for rendering components, you dont want to be hidden, clipped by parent component. For example, models, popovers, dropdowns.
+
+## Table of contents
+
+- [Import](#import)
+- [Examples](#examples)
+  - [Basic usage](#basic-usage)
+- [Props](#props)
+- [Help improve this page](#help-improve-this-page)
+
+## Import
+
+```jsx static=true
+import { Portal } from '@contentful/f36-utils';
+```
+
+## Example
+
+### Basic usage
+
+```jsx static=true
+<Box>
+  <Portal>
+    <Box
+      style={{
+        backgroundColor: 'gray',
+      }}
+    >
+      Hi! I'm first in the document flow, but will be rendered at the end of the
+      document!
+    </Box>
+  </Portal>
+  Parent component content
+</Box>
```

Comment:
I don't understand the Box component here

Shouldn't it be a Fragment or a Flex?